pyvista/examples/examples.py

```diff
@@ -412,3 +412,65 @@ def load_nut():
 
     """
     return pyvista.read(os.path.join(dir_path, 'nut.ply'))
+
+
+def _cart2sphe(x, y, z):
+    """Convert 3D Cartesian coordinates to spherical coordinates.
+
+    Parameters
+    ----------
+    x, y, z : np.ndarray
+        Coordinates.
```

Comment:
Will numpydoc accept this? Or are we using the fact that this won't have docs built?